public/pages/utils/anomalyResultUtils.ts

```diff
@@ -584,7 +604,23 @@ export const parseBucketizedAnomalyResults = (result: any): Anomalies => {
           endTime: get(rawAnomaly, 'data_end_time'),
           plotTime: get(rawAnomaly, 'data_end_time'),
           entity: get(rawAnomaly, 'entity'),
+          contributions: toFixedNumberForAnomaly(
+            get(rawAnomaly, 'anomaly_grade')
+          ) > 0 ? contributions : {}
+        });
+
+        const featureBreakdownAttributionMap: {[key: string]: number} = {};
+
+        get(rawAnomaly, 'relevant_attribution', []).filter(element => get(element, 'feature_id') && get(element, 'data')).
+        forEach((attribution: any) => {
+          featureBreakdownAttributionMap[get(attribution, 'feature_id')] = toFixedNumberForAnomaly(get(attribution, 'data'));
+        });
+             
+        const expectedValueMap: {[key: string]: number} = {};
+        get(rawAnomaly, 'expected_values[0].value_list', []).forEach((expect: any) => {
```

Comment:
can the `forEach` arg be renamed and typesafe? For example, `expectedValue: number`?

Reply:
sure, will refactor this part later